model/AbstractRdsResultStorage.php

```diff
@@ -0,0 +1,727 @@
+        $object->callIdTest = $variable[self::CALL_ID_TEST_COLUMN];
+        $object->test = $variable[self::TEST_COLUMN];
+        $object->item = $variable[self::ITEM_COLUMN];
+        $object->variable = clone $resultVariable;
+
+        return $object;
+    }
+
+    /**
+     * @return QueryBuilder
+     */
+    protected function getQueryBuilder()
+    {
+        return $this->getPersistence()->getPlatform()->getQueryBuilder();
+    }
+
+    /**
+     * @return \common_persistence_SqlPersistence
+     */
+    public function getPersistence()
+    {
+        if ($this->persistence === null) {
+            $persistenceId = $this->hasOption(self::OPTION_PERSISTENCE) ?
+                $this->getOption(self::OPTION_PERSISTENCE)
+                : 'default';
+            $this->persistence = $this->getServiceLocator()->get(PersistenceManager::SERVICE_ID)->getPersistenceById($persistenceId);
+        }
+
+        return $this->persistence;
+    }
+
+    /**
+     * @param $value
+     *
+     * @return mixed
+     */
+    protected function unserializeVariableValue($value)
+    {
+        return unserialize($value);
+    }
+
+    /**
+     * @param $value
+     *
+     * @return string
+     */
+    protected function serializeVariableValue($value)
+    {
+        return serialize($value);
+    }
+
+    /**
+     * @param array $data
+     *
+     * @throws DuplicateVariableException
+     */
+    private function insertMultiple(array $data)
+    {
+        $duplicatedData = false;
+        try {
+            $this->getPersistence()->insertMultiple(self::VARIABLES_TABLENAME, $data);
+        } catch (UniqueConstraintViolationException $e) {
+            $duplicatedData = true;
+            foreach ($data as $row) {
+                try {
+                    $this->getPersistence()->insert(self::VARIABLES_TABLENAME, $row);
+                } catch (UniqueConstraintViolationException $e) {
+                    //do nothing, just skip it
+                }
+            }
+        }
+
+        if ($duplicatedData) {
+            throw new DuplicateVariableException(sprintf('An identical result variable already exists.'));
+        }
+    }
+
+    public function spawnResult()
+    {
+        \common_Logger::w('Unsupported function');
+    }
+
+    /*
+     * retrieve specific parameters from the resultserver to configure the storage
+     */
+    public function configure($callOptions = [])
+    {
+        \common_Logger::d('configure  RdsResultStorage with options : ' . implode(" ", $callOptions));
+    }
+
+    /**
+     *
+     * @param mixed $a
+     * @param mixed $b
+     *
+     * @return number
+     */
+    public static function sortTimeStamps($a, $b)
+    {
+        list($usec, $sec) = explode(" ", $a);
+        $floata = ((float)$usec + (float)$sec);
+        list($usec, $sec) = explode(" ", $b);
+        $floatb = ((float)$usec + (float)$sec);
+        //common_Logger::i($a." ".$floata);
```

Comment:
please remove comments,
if needed we can retrieve the code from git

Reply:
Fixed